guides/source/action_view_overview.md

```diff
@@ -1505,6 +1505,266 @@ strip_tags("<b>Bold</b> no more!  <a href='more.html'>See more</a>")
+ the value of the link itself will become the name.
+
+**Examples**
+
+Because it relies on `url_for`, `link_to` supports both older-style `controller/action/id` arguments
+and newer RESTful routes. Current Rails style favors RESTful routes whenever possible, so base
+your application on resources and use
+
+```ruby
+link_to "Profile", profile_path(@profile)
+# => <a href="/profiles/1">Profile</a>
+```
+
+or the even pithier
+
+```ruby
+link_to "Profile", @profile
+# => <a href="/profiles/1">Profile</a>
+```
+
+in place of the older more verbose, non-resource-oriented
+
+```ruby
+link_to "Profile", controller: "profiles", action: "show", id: @profile
+# => <a href="/profiles/show/1">Profile</a>
+```
+
+Similarly,
+
+```ruby
+link_to "Profiles", profiles_path
+# => <a href="/profiles">Profiles</a>
+```
+
+is better than
+
+```ruby
+link_to "Profiles", controller: "profiles"
+# => <a href="/profiles">Profiles</a>
+```
+
+When name is `nil` the href is presented instead
+
+```ruby
+link_to nil, "http://example.com"
+# => <a href="http://www.example.com">http://www.example.com</a>
+```
+
+You can use a block as well if your link target is hard to fit into the name parameter. ERB example:
+
+```html+erb
+<%= link_to(@profile) do %>
+    <strong><%= @profile.name %></strong> -- <span>Check it out!</span>
+<% end %>
+# => <a href="/profiles/1">
+        <strong>David</strong> -- <span>Check it out!</span>
+     </a>
+```
+See [API Doc for more info](https://api.rubyonrails.org/v6.0/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to)
+
+#### button_to
+Generates a form containing a single button that submits to the URL created
+by the set of `options`. This is the safest method to ensure links that
+cause changes to your data are not triggered by search bots or accelerators.
+If the HTML button does not work with your layout, you can also consider
+using the `link_to` method with the `:method` modifier as described in
+the `link_to` documentation.
+
+##### Data attributes
+
+* `:confirm` - This will use the unobtrusive JavaScript driver to
+prompt with the question specified. If the user accepts, the link is
+processed normally, otherwise no action is taken.
+* `:disable_with` - Value of this parameter will be
+used as the value for a disabled version of the submit
+button when the form is submitted. This feature is provided
+by the unobtrusive JavaScript driver.
+
+##### Examples
+
+```html+erb
+<%= button_to "New", action: "new" %>
+# => "<form method="post" action="/controller/new" class="button_to">
+#      <input value="New" type="submit" />
+#    </form>"
+
+<%= button_to "New", new_article_path %>
+# => "<form method="post" action="/articles/new" class="button_to">
+#      <input value="New" type="submit" />
+#    </form>"
+
+```
+See [API Doc for more info](https://api.rubyonrails.org/v6.0/classes/ActionView/Helpers/UrlHelper.html#method-i-button_to)
+
+#### link_to_unless_current
```

Comment:
This isn't necessary to document here. Please remove this line to the end of `phone_to`.